.env.example

```diff
@@ -1,5 +1,4 @@
 UBERDUCK_API_HOST=localhost:8000/audio/response
-ML_API_URL=http://openduck_ml_1:8001
```

Comment:
let's keep this. Set it to `ML_API_URL=http://localhost:8000` by default

it's a good abstraction for when they need to be scaled independently